_download-meta/openttd-releases.md

```diff
@@ -2,9 +2,13 @@
 folder_old_infrastructure: releases
 ---
 
-For OpenTTD you can use the original Transport Tycoon Deluxe data files (you need to own a Transport Tycoon Deluxe CD).
-There are also the free alternatives: [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx), [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx).
-These can be installed automatically by the Windows and OS/2 installers.
-Please refer to the [readme](https://proxy.binaries.openttd.org/openttd-releases/@@version@@/README.md) for more information.
+OpenTTD requires one version of base graphics; available for installation with the installer are:
+* [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx)
+* Transport Tycoon Deluxe data files can be installed if you own the Transport Tycoon Deluxe CD
 
-You can download the free alternatives here: [download OpenGFX](../opengfx-releases/latest.html), [download OpenSFX](../opensfx-releases/latest.html), and [download OpenMSX](../openmsx-releases/latest.html).
+Sound and music sets are recommended (but not necessary).
+[OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx) can be installed via the installer or - if you own the CD -  also the Transport Tycoon Deluxe sound and music.
+
+Additional graphics, sound and music sets can be downloaded via the ingame content download manager.
+
+Please refer to the [readme](https://binaries.openttd.org/releases/@@version@@/readme.txt) for more information.
```

Comment:
This URL changed to `[readme](https://proxy.binaries.openttd.org/openttd-releases/@@version@@/README.md)`